apis/v1alpha1/gameserverset_types.go

```diff
@@ -139,6 +139,9 @@ type GameServerSetStatus struct {
 	WaitToBeDeletedReplicas *int32 `json:"waitToBeDeletedReplicas,omitempty"`
 	// LabelSelector is label selectors for query over pods that should match the replica count used by HPA.
 	LabelSelector string `json:"labelSelector,omitempty"`
+	// The generation observed by the controller.
+	// +optional
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
```

Comment:
Please move the ObservedGeneration to 131 line.

Reply:
Done